src/DB/projects.json

```diff
@@ -1096,5 +1096,23 @@
         "tech": ["Javascript", "Css", "Html"]
       }
     ]
+  },
+  {
+    "github_username": "YOUR_GITHUB_USERNAME",
+    "Social_media": {
+      "gitHub": "https://github.com/Sriparno08",
+      "LinkedIn": "https://www.linkedin.com/in/sriparnoroy/",
+      "Instagram": "",
+      "YouTube": "",
+      "Twitter": "https://twitter.com/Sriparno08"
+    },
+    "Projects": [
+      {
+        "link": "https://github.com/Sriparno08/Start-Contributing",
+        "title": "Start Contributing",
+        "description": "This project aims to help beginners get started with contributing to open source by providing a substantial amount of useful resources. Everyone is welcome to contribute and be a part of this project.",
+        "tech": ""
```

Comment:
add tech stacks that you have used to make this project

Reply:
Hey @priyankarpal, this project is just a repository that contains open-source resources. I haven't really used anything that I can include in my tech stack. So, what should I put here?

Reply:
Okay 👍